libpod/oci.go

```diff
@@ -399,15 +400,18 @@ func (r *OCIRuntime) updateContainerStatus(ctr *Container) error {
 	// Store old state so we know if we were already stopped
 	oldState := ctr.state.State
 
-	out, err := exec.Command(r.path, "state", ctr.ID()).Output()
+	out, err := exec.Command(r.path, "state", ctr.ID()).CombinedOutput()
 	if err != nil {
+		if strings.HasSuffix(string(out), "does not exist") {
+			ctr.removeConmonFiles()
+			ctr.state.State = ContainerStateConfigured
+			return nil
```

Comment:
We should probably still return `err` here - the first time we run `ps` will error, but if we try it again it will succeed because the container is known to be dead.

Reply:
Actually, wait, we might not do a save() if this errors.

Reply:
Yeah, we only force a save if there's no error here. Safest to return nil then.

Reply:
I just want to be able to do a podman rm UUID, which this should enable.

Reply:
I think this should be enough to put it back in a state where you can actually use it.